rskj-core/src/main/java/co/rsk/peg/bitcoin/UtxoUtils.java

```diff
@@ -0,0 +1,78 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.bitcoinj.core.VarInt;
+import com.google.common.primitives.Bytes;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.spongycastle.util.encoders.Hex;
+
+public class UtxoUtils {
+    private UtxoUtils() {}
+
+    /**
+     * Decode a {@code byte[]} of encoded outpoint values.
+     *
+     * @param encodedOutpointValues
+     * @return {@code List<Coin>} the list of outpoint values decoded in the same ordered they were before encoding.
+     * Or an {@code Collections.EMPTY_LIST} when {@code encodedOutpointValues} is {@code null} or {@code empty byte[]}.
+     */
+    public static List<Coin> decodeOutpointValues(byte[] encodedOutpointValues){
```

Comment:
Something interesting could be to also have either a public or private `decodeOutpointValue` method that gets called from this method. This can lead to this method being shorter. And if it is public maybe it can be used in other places in the future where only one outpoint value is needed. wdyt?

Reply:
Nice idea. Let's discuss it further to see its benefits.

Reply:
I will mark it this as resolved, and leave it for you to decide ;)